src/ClientData/CaptureData.cpp

```diff
@@ -93,7 +93,7 @@ void CaptureData::AddScopeStats(ScopeId scope_id, ScopeStats stats) {
 
 void CaptureData::OnCaptureComplete() {
   thread_track_data_provider_->OnCaptureComplete();
-  all_scopes_->SortTimers();
+  all_scopes_->OnDataChanged();
```

Comment:
`OnDataChanged()` looks as something that will be executed every time the data changes. 

Maybe `OnDataChangesComplete()`? But TBH I would prefer `OnCaptureComplete()` just because consistency.